src/sentry/conf/server.py

```diff
@@ -2752,7 +2743,7 @@ def build_cdc_postgres_init_db_volume(settings: Any) -> dict[str, dict[str, str]
                 "CLICKHOUSE_HTTP_PORT": "8123",
                 "DEFAULT_BROKERS": ""
                 if "snuba" in settings.SENTRY_EVENTSTREAM
-                else "{containers[kafka][name]}:9093",
+                else "{containers[kafka][name]}:9092",
```

Comment:
I think this can/should go back to 9093 now that I fixed the internal access

Reply:
works either way (tested via `tests/snuba/incidents/test_tasks.py`), but yeah let's prefer internal access